src/app/layouts/base-layout/components/header/nav-item/nav-item.tsx

```diff
@@ -37,11 +56,15 @@ export const NavItem = ({ label, href, dropdownInner }: NavItemProps) => {
         onMouseEnter={onOpen}
         end
       >
-        <p className={cx('label')}>{label}</p>
+        <span className={cx('label')}>{label}</span>
         {dropdownInner && (
-          <span className={cx('dropdown-arrow')}>
+          <button
+            onKeyDown={handleKeyPress}
+            className={cx('dropdown-arrow')}
+            aria-expanded={isDropdownOpen}
+          >
```

Comment:
Maybe we should make the focus state more visible, because users with poor vision might not notice it
![image](https://github.com/user-attachments/assets/4c0d257a-f41e-42c2-9669-09c8923231b9)

Reply:
I'v added some outline-offset to it to be more visible.